beets/ui/commands.py

```diff
@@ -1450,7 +1450,7 @@ def modify_func(lib, opts, args):
 
 # move: Move/copy files to the library or a new base directory.
 
-def move_items(lib, dest, query, copy, album, pretend, confirm=False):
+def move_items(lib, dest, query, copy, album, pretend, confirm=False, export):
```

Comment:
Can you please be a little more specific? I can't quite find the error you're talking about; maybe you can include the error text?